plugins/scaffolder-backend/src/scaffolder/jobs/types.ts

```diff
@@ -31,7 +31,7 @@ export type StageContext<T = {}> = {
 
 export type ProcessorStatus = 'PENDING' | 'STARTED' | 'COMPLETED' | 'FAILED';
 
-export interface Stage extends StageInput {
+export interface StageResult extends StageInput {
```

Comment:
Maybe more symmetrical with `StageOutput`, but no strong opinion at all

Reply:
Hah - it was @Rugvip that made me change it from `StageOutput` to this.

Reply:
https://github.com/spotify/backstage/pull/1437#discussion_r446875375

Reply:
Ah wait, now I realise you might have already seen the previous comment. 🏄🏻‍♂️

Reply:
I cannot fathom what I'm being accused of in this thread. I merely suggested a rename from `Stage` to some other shite